src/Native/Windows/clrcompression/zlib-intel/deflate.c

```diff
@@ -1,5 +1,5 @@
 /* deflate.c -- compress data using the deflation algorithm
- * Copyright (C) 1995-2013 Jean-loup Gailly and Mark Adler
+ * Copyright (C) 1995-2017 Jean-loup Gailly and Mark Adler
```

Comment:
I'm curious, what causes some of these copyright dates to differ, e.g. https://github.com/dotnet/corefx/pull/32732/files#diff-d1964061700ad02918d1ccd83e7e4a86R2 being updated to 2016 and this being updated to 2017?

Reply:
The copyright headers (including the dates) were retained from the upstream ZLib for existing files that we modified.